packages/strapi-plugin-users-permissions/controllers/Auth.js

```diff
@@ -395,7 +395,7 @@ module.exports = {
     }
 
     const params = {
-      ..._.pick(ctx.request.body, ['username', 'email', 'password']),
+      ..._.omit(ctx.request.body, ['confirmed']),
```

Comment:
You should also omit the resetPasswordToken

Reply:
updated !